Remotion/Web/ClientScript/Scripts/DropDownMenu.ts

```diff
@@ -731,34 +731,21 @@ class DropDownMenu
     const anchor = document.createElement("a");
     anchor.setAttribute('role', 'menuitem');
     anchor.setAttribute('tabindex', '-1');
-    if (isEnabled)
+    if (isEnabled && itemInfo.Href != null)
```

Comment:
I would just use the value for the truthy check and remove the explict null check. Only thing to remember is that an empty string is also falsey but in this case I don't think that check could be good anyways.

```suggestion
    if (isEnabled && itemInfo.Href)
```

If you want to keep the explicit check please use `!==` to prevent type conversion issues.